packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -11,9 +11,14 @@ import { ODS_TOOLTIP_STRATEGY, type OdsTooltipStrategy } from '../../constants/t
 export class OdsTooltip {
   private arrowElement?: HTMLElement;
   private triggerElement?: HTMLElement | null;
+  private isTooltipHover: boolean = false;
+  private timer?: NodeJS.Timeout;
```

Comment:
Not a huge fan of using a NodeJS typing here, maybe try https://stackoverflow.com/a/56239226 ?